tools/compare_import_time.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2024-Present MongoDB, Inc.
```

Comment:
Should this tool live in the test/ dir instead?

Reply:
I don't think so, since we won't run it as part of `pytest`.

Reply:
Sounds good.